nginx-stable/snapcraft.yaml

```diff
@@ -0,0 +1,63 @@
+name: nginx
+version: "1.11.1"
+summary: nginx is an HTTP and reverse proxy server
+description: |
+  nginx ("engine x") is an HTTP and reverse proxy server, a mail proxy server,
+  and a generic TCP/UDP proxy server, originally written by Igor Sysoev.  For a
+  long time, it has been running on many heavily loaded Russian sites including
+  Yandex, Mail.Ru, VK, and Rambler.
+confinement: devmode
+
+apps:
+  nginx:
+    command: usr/nginx/nginx
+    stop-command: nginx -s quit
+    daemon: simple
+    plugs: [network, network-bind]
+
+parts:
+  nginx:
+    source: http://hg.nginx.org/nginx/archive/default.tar.gz
+    plugin: autotools_nginx
+    configflags:
+      - --sbin-path=/usr/nginx/nginx
+      - --conf-path=/usr/nginx/nginx.conf
+      - --pid-path=/usr/nginx/nginx.pid
+      - --error-log-path=/log/error.log
+      - --http-log-path=/log/access.log
```

Comment:
The absolute aspect was a mistake, this was experimenting with the strange way that `nginx`'s build process stringifies the compile time arguments to set paths (meaning that it permanently hardcodes things to look in the build directory as "/home/user/snappy-playpen/nginx/stage/log/error.log" as an example)

Thanks to your comments I've successfully snapped the application, but configuration is a million miles away from making sense; I worry that it will need the level of expertise that kyrofa brought to snapping `apache2` at least (hacking the `./configure` script and the snapcraft plugin, etc)